packages/docs/docs/docs/deploying.md

```diff
@@ -178,6 +178,16 @@ to understand is that the contracts deployed by the `push` command are logic con
 upgradeable instances, as we will see later in the
 [Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
 
+For example, you wouldn't interact with the instance of MyContract that we just deployed. This instance is only intended for defining the behavior or logic of the contract. You can find the address of this logic contract in the `zos.dev-<network_id>.json` file, under `contracts.MyContract.address`. Remember to never interact with this address!
```

Comment:
```suggestion
For example, you wouldn't interact with the instance of MyContract that we just deployed. This instance is only intended for defining the behavior or logic of the contract. You can find the address of this logic contract in the `zos.dev-<network_id>.json` file, under `contracts.MyContract.address`.
```

I think the point is already clear :-)

Reply:
Also, I'd avoid referring to the logic contract as a "MyContract instance", since it could be confused with the "upgradeable instance". I'd just say "would not interact with the MyContract that we just deployed", etc.

Reply:
> I'd avoid referring to the logic contract as a "MyContract instance", since it could be confused with the "upgradeable instance". I'd just say "would not interact with the MyContract that we just deployed", etc.

I think this part got hidden when accepting the suggestion!